ecosystem/typescript/sdk_v2/src/client/get.ts

```diff
@@ -71,9 +71,7 @@ export async function get<Req, Res>(options: GetRequestOptions): Promise<AptosRe
   return response;
 }
 
-export async function getAptosFullNode<Req, Res>(
-  options: GetAptosRequestOptions,
-): Promise<AptosResponse<Req, Res>> {
+export async function getAptosFullNode<Req, Res>(options: GetAptosRequestOptions): Promise<AptosResponse<Req, Res>> {
```

Comment:
I did run `pnpm fmt` and `pnpm lint` btw, the only reason this is changed is because I ran it :p

the linting errors that remain are from the public key interface not being complete (I think)